### ISSUE TITLE
Missing api check

### ISSUE THREAD

**Issue by MarcAnt01** [Magisk-Modules-Repo/miracast-enabler#3]
You said that the module is only for Android Pie 9.0, but there is no api check in install.sh

**Comment by crass** [comment]
Yeah, I left out the API check so that users or other versions could try it out and see if it worked (mostly likely not).

**Comment by MarcAnt01** [comment]
@crass I understand your need of testing, but all the stuff on Magisk Repo should be tested before being pushed to all users; not everyone is able to come out of a bootloop and there are many noobs
